src/site/content/en/angular/angular.11tydata.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
           'get-started-optimize-angular',
         ],
       },
+      {
```

Comment:
btw I noticed this 11tydata file has `react` in the tags at the top :)

Reply:
I'll fix this in the next PR.